### PR TITLE
Add basic link in bio intro step

### LINKED ISSUE #66085: Signup: Add Link in Bio initial step
### Details

This adds the initial step for Link in Bio flow.
The Flow should be available on `setup/linkInBioIntro?flow=link-in-bio`.

Note: This is only related to the very first step of the flow

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by renancarvalho**
#### Proposed Changes
Add basic initial link in bio step

#### Testing Instructions
 - Pull this branch and Run `yarn start`
 - Navigate to http://calypso.localhost:3000/setup/linkInBioIntro?flow=link-in-bio

You should see a page very similar to this:

image

Related to #66085
Fixes #66085

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-intro/intro.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	goNext: () => void;
+}
+
+const Intro: React.FC< Props > = ( { goNext } ) => {
+	return (
+		<div className="link-in-bio-intro__content">
+			<h1 className="link-in-bio-intro__title">
+				Hello!
+				<span>Let’s set up your</span>
+				Link in Bio
```
Comment (escapemanuele) [inline client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-intro/intro.tsx]: Remember about translations, look at how `@wordpress/react-i18n` is used in other steps 😄

File: client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-intro/intro.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+			</h1>
+			<Button className="link-in-bio-intro__button" primary onClick={ goNext }>
+				Get Started
```
Comment (escapemanuele) [inline client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-intro/intro.tsx]: Here as well

**Review by escapemanuele — CHANGES_REQUESTED** [review]
Well done @renancarvalho 🚀 🎸 
While we see how to deal with the large image, could be nice to fix translations 😄

**Review by escapemanuele — APPROVED** [review]
Wonderful, thanks for making the changes!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7459574

Thank you @renancarvalho for including a screenshot in the description! This is really helpful for our translators.

**Comment by emilyaudela** [comment]
Hi there! We got a note from one of our translators that it's hard to translate this phrase because it's been separated into 2 [separate strings](https://translate.wordpress.com/deliverables/de/7459574/). Would it be possible to put the two parts in one string, instead of splitting it into 2 strings? So it would read `<span>Let’s set up your</span> Link in Bio`.

**Comment by renancarvalho** [comment]
> Hi there! We got a note from one of our translators that it's hard to translate this phrase because it's been separated into 2 [separate strings](https://translate.wordpress.com/deliverables/de/7459574/). Would it be possible to put the two parts in one string, instead of splitting it into 2 strings? So it would read `<span>Let’s set up your</span> Link in Bio`.

Hi @emilyaudela, thank you for getting in touch, Yes, this is being handled on this PR https://github.com/Automattic/wp-calypso/pull/66102 which I believe will be merged very soon.
Thank you.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7459574).

**Pull request merged**